tests/orm/nodes/data/test_kpoints.py

```diff
@@ -82,3 +83,9 @@ def test_get_kpoints(self):
         kpt2 = load_node(kpt.pk)
         assert np.abs(kpt2.get_kpoints() - np.array(kpoints)).sum() == 0.
         assert np.abs(kpt2.get_kpoints(cartesian=True) - np.array(cartesian_kpoints)).sum() == 0.
+
+    def test_init(self):
+        """Test that the setters properties of `KpointsData` can be defined via the constructor."""
+        kpt = KpointsData(mesh=self.mesh)
+
+        assert kpt.get_kpoints_mesh()[0] == self.mesh
```

Comment:
I think this is not correct, if you look at method `set_kpoints_mesh` the offset is set at the same time when given mesh. The mesh and offset looks always appear with each other, you need to consider it. For the test, it is easier to test both of them by passing `print_list=True`.